lms/static/sass/views/_shoppingcart.scss

```diff
@@ -56,6 +56,8 @@ $light-border: 1px solid $gray-l5;
 }
 
 .confirm-enrollment {
+  @include direction();
```

Comment:
Actually, I found this in https://github.com/edx/edx-platform/blob/master/lms/static/sass/vendor/bi-app/_mixins.scss#L268. I don't think it has anything to do with bourbon, but I might be wrong. It is part of the bi-app, which is included in the mako template that's used to compile into sass.

To answer @marcotuts's question:

The reason for this is the issue reported in the [UX-2261!](https://openedx.atlassian.net/browse/UX-2261) ticket. This seemed to fix the issue of direction not being set correctly. By explicitly setting the direction, I've manage to solve the issue for the shopping cart.

@frrrances, I'll look into alternative way to achieve a desired effect, but if this isn't part of bourbon, would it be ok to use it, since it seems like a bi-app way to set a direction?

Thanks!